src/com/company/Main.java

```diff
@@ -67,9 +88,16 @@ public static void searchTheWord(String data, String searchword) {
         }
         if (count == 0) {
             System.out.println("Word not found");
+            String errorMessage = "Word not found";
```

Comment:
Word not found string is used in line 90 and 91, so make the text a static final constant

Reply:
@premsrj can we have a properties file to keep these messages? Or will that be out of scope here?

Reply:
Better to add constant class and keep all the message to this class

Reply:
kept all the constants in the separate class and implemented.